atflatcontrols/atstatusbar.pas

```diff
@@ -632,11 +656,8 @@ procedure TATStatus.DoPanelStretch(AIndex: integer);
     if i<>AIndex then
     begin
       D:= GetPanelData(i);
-      NCell:= D.Width;
-      if not D.AutoSize then
-        NCell:= Theme^.DoScale(NCell);
       if Assigned(D) then
-        Inc(NSize, NCell);
+        Inc(NSize, D.Width);
     end;
```

Comment:
NCell lost

Reply:
? Remove? Or...?

Reply:
I added NCell today, just get my fix from GH and don’t change

Reply:
You'll have to tell me how I do that. Just download the file?

Reply:
Download the file and copy paste my NCell block inside your patch so diff won’t show the change

Reply:
Done.

Reply:
Still getting an identifier not found "LazarusResources" error when compiling the lazarus package. I think that's the only error that one is throwing.